scripts/build.js

```diff
@@ -379,7 +379,9 @@ if (isDeveloping) {
         return;
       }
 
-      await regenerateBundle();
+      if (filename.includes('src/') && /\.(js|ts)$/.test(filename)) {
+        await regenerateBundle();
+      }
```

Comment:
I think this will cause stale components on `.css` files

```suggestion
      if (filename.includes('src/') && /\.(js|ts|css)$/.test(filename)) {
        await regenerateBundle();
      }
```